backend/src/main/java/org/cryptomator/hub/api/LicenseResource.java

```diff
@@ -0,0 +1,49 @@
+package org.cryptomator.hub.api;
+
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.cryptomator.hub.entities.EffectiveVaultAccess;
+import org.cryptomator.hub.license.LicenseHolder;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+
+import java.time.Instant;
+import java.util.Optional;
+
+@Path("/license")
+public class LicenseResource {
+
+	@Inject
+	LicenseHolder licenseHolder;
+
+	@GET
+	@Path("/status")
+	@Produces(MediaType.APPLICATION_JSON)
+	@RolesAllowed("user")
+	@Operation(summary = "Get license status information", description = "Information includes the license expiration date, the licensed seats and the already used seats")
```

Comment:
```suggestion
	@Operation(summary = "Get license status information", description = "Information includes the license expiration date (optional), the licensed seats and the already used seats")
```

Reply:
Fixed in f9c3f00f7f1d5197a054455571229cbe1ae3e716.